<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lobby</class>
 <widget class="QWidget" name="lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:none;
</string>
  </property>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>345</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Login/Create an account&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgba(0, 0, 0, 148);
	border: none;
}

QPushButton:hover{
	background-color: rgb(190, 190, 190);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 170);
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameStartBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Play Tic Tac Toe!&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgba(0, 0, 0, 148);
	border: none;
}

QPushButton:hover{
	background-color: rgb(190, 190, 190);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 170);
}</string>
   </property>
   <property name="text">
    <string>Game Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaderboardsBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>475</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Check your rank!&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgba(0, 0, 0, 148);
	border: none;
}

QPushButton:hover{
	background-color: rgb(190, 190, 190);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 170);
}</string>
   </property>
   <property name="text">
    <string>Leaderboards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Adjust the background music!&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgba(0, 0, 0, 148);
	border: none;
}

QPushButton:hover{
	background-color: rgb(190, 190, 190);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 170);
}</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>605</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Quit the game&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgba(0, 0, 0, 148);
	border: none;
}

QPushButton:hover{
	background-color: rgb(190, 190, 190);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 170);
}</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>120</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0,0,0,148);
background:white;</string>
   </property>
   <property name="text">
    <string>Tic Tac Toe</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/tictactoe.png);
                    background-repeat: no-repeat;
                    background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>loginBtn</zorder>
  <zorder>gameStartBtn</zorder>
  <zorder>leaderboardsBtn</zorder>
  <zorder>settingBtn</zorder>
  <zorder>quitBtn</zorder>
  <zorder>title</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
